google/cloud/spanner_v1/client.py

```diff
@@ -314,6 +315,10 @@ def instance(
         :param node_count: (Optional) The number of nodes in the instance's
                             cluster; used to set up the instance's cluster.
 
+        :type processing_units: int
+        :param processing_units: (Optional) The number of processing units
+                                allocated to this instance.
```

Comment:
Might be worth mentioning that you can only set either `node_count` or `processing_units`